### PR TITLE
Remove � character

### PR DESCRIPTION
**Pull request by roukmoute**

### CODE REVIEW

File: ibm_db2/ibm_db2.php [hunk 1]
```diff
@@ -139,7 +139,7 @@ function db2_autocommit ($connection, $value = null) {}
  * from stored procedures. A non-negative value specifies the maximum size of 
  * the XML data that will be retrieved from the database. If this parameter 
  * is not used, a default of 1MB will be assumed for retrieving the XML 
- * output value from the stored procedure.�
+ * output value from the stored procedure.
```
Comment (andrey-sokolov) [inline ibm_db2/ibm_db2.php]: Please try not to include such changes

Comment (andrey-sokolov) [inline ibm_db2/ibm_db2.php]: Please, try not to include such changes

Comment (andrey-sokolov) [inline ibm_db2/ibm_db2.php]: Please, try not to include such changes

File: libevent/libevent.php [hunk 2]
```diff
@@ -52,7 +52,7 @@
  * If you want to make it non-pending from within its callback, you can call
  * event_del() on it.
  *
- * The timeout on a persistent event resets whenever the event�s callback runs.
+ * The timeout on a persistent event resets whenever the events callback runs.
```
Comment (andrey-sokolov) [inline libevent/libevent.php]: event's

File: libevent/libevent.php [hunk 3]
```diff
@@ -151,7 +151,7 @@ function event_base_free($event_base) {}
  * <p>By default, the {@link event_base_loop}() function runs an event_base until
  * there are no more events registered in it. To run the loop, it repeatedly
  * checks whether any of the registered events has triggered (for example,
- * if a read event�s file descriptor is ready to read, or if a timeout event�s
+ * if a read events file descriptor is ready to read, or if a timeout events
```
Comment (andrey-sokolov) [inline libevent/libevent.php]: event's

File: libevent/libevent.php [hunk 4]
```diff
@@ -543,12 +543,12 @@ function event_buffer_timeout_set($bevent, $read_timeout, $write_timeout) {}
...
  *
  * <p><b>Read low-water mark</b><br/>
- * Whenever a read occurs that leaves the bufferevent�s input buffer at this
- * level or higher, the bufferevent�s read callback is invoked. Defaults to 0,
+ * Whenever a read occurs that leaves the bufferevents input buffer at this
+ * level or higher, the bufferevents read callback is invoked. Defaults to 0,
  * so that every read results in the read callback being invoked.</p>
  *
  * <p><b>Read high-water mark</b><br/>
- * If the bufferevent�s input buffer ever gets to this level, the bufferevent
```
Comment (andrey-sokolov) [inline libevent/libevent.php]: bufferevent's

File: libevent/libevent.php [hunk 5]
```diff
@@ -183,7 +183,7 @@ function event_base_loop($event_base, $flags = null) {}
  *
  * <p>It differs from {@link event_base_loopexit}() in that if the event_base is currently
  * running callbacks for any active events, it will exit immediately after finishing the
- * one it�s currently processing. The behaviour is similar to break statement.</p>
```
Comment (andrey-sokolov) [inline libevent/libevent.php]: it's

**Comment by andrey-sokolov** [comment]
Thank you for noticing this problem. We will fix it correctly

**Comment by roukmoute** [comment]
Oh I'm sorry I haven't corrected your requests!
I am generally very careful to be responsive and I have not seen the first notification when you have made your remarks.

Would you like I make a new clean P.R.?

**Comment by andrey-sokolov** [comment]
@roukmoute We are glad to accept your changes with required corrections. :) I've reopened the pull request

File: ibm_db2/ibm_db2.php [hunk 6]
```diff
@@ -139,7 +139,7 @@ function db2_autocommit ($connection, $value = null) {}
  * from stored procedures. A non-negative value specifies the maximum size of 
  * the XML data that will be retrieved from the database. If this parameter 
  * is not used, a default of 1MB will be assumed for retrieving the XML 
- * output value from the stored procedure.�
```
Comment (roukmoute) [inline ibm_db2/ibm_db2.php]: I know this is really strange but I've got a special char with my IDE, this is why I remove this char.
special_char

Comment (andrey-sokolov) [inline ibm_db2/ibm_db2.php]: That's my mistake, you right, this change is good

**Pull request merged**